### DISCUSSION TITLE
Upgrade old drivers

### DISCUSSION THREAD

**Discussion by git2013vb** [NVIDIA/open-gpu-kernel-modules#438] [Q&A] [answered]
Hi,
I have a old NVIDIA card (GeForce GT 650M) in a old laptop (Acer).
he video card is not supported by NVIDIA any more so I'm using the legacy drivers (390.157)

Can I use this driver? Is there a  branch compatible?

Thanks

**Comment by aritger** [comment, marked as answer]
Sorry, no.  The open-gpu-kernel-modules git repo only supports Turing and later GPUs listed here: https://github.com/NVIDIA/open-gpu-kernel-modules#readme

There is no support for legacy GPUs, sorry.
